### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by ytetra**
fix typo

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/katib/233)

### CODE REVIEW

File: cmd/cli/command/push-model.go [hunk 1]
```diff
@@ -64,13 +64,13 @@ func pushModel(cmd *cobra.Command, opt *pushModelOpt) {
 		buf, _ := ioutil.ReadFile(opt.file)
 		err = yaml.Unmarshal(buf, &req)
 		if err != nil {
-			log.Fatalf("Fail to Purse config: %v", err)
+			log.Fatalf("Fail to Parse config: %v", err)
```
Comment (gaocegege) [inline cmd/cli/command/push-model.go]: s/Parse/parse

File: cmd/cli/command/push-model.go [hunk 2]
```diff
@@ -64,13 +64,13 @@ func pushModel(cmd *cobra.Command, opt *pushModelOpt) {
...
 			return
 		}
 	} else if len(opt.args) > 0 {
 		err := json.Unmarshal(([]byte)(opt.args[0]), &req)
 		if err != nil {
-			log.Fatalf("Fail to Purse input: %v", err)
+			log.Fatalf("Fail to Parse input: %v", err)
```
Comment (gaocegege) [inline cmd/cli/command/push-model.go]: s/Parse/parse

File: cmd/cli/command/push-model.go (hunk 1 above)
Comment (ytetra) [reply cmd/cli/command/push-model.go]: thanks, fixed

File: cmd/cli/command/push-model.go (hunk 2 above)
Comment (ytetra) [reply cmd/cli/command/push-model.go]: fixed

File: cmd/cli/command/push-study.go [hunk 3]
```diff
@@ -65,13 +65,13 @@ func pushStudy(cmd *cobra.Command, opt *pushStudyOpt) {
 		buf, _ := ioutil.ReadFile(opt.file)
 		err = yaml.Unmarshal(buf, &in)
 		if err != nil {
-			log.Fatalf("Fail to Purse config: %v", err)
+			log.Fatalf("Fail to Parse config: %v", err)
```
Comment (gaocegege) [inline cmd/cli/command/push-study.go]: Same here

File: cmd/cli/command/push-study.go [hunk 4]
```diff
@@ -65,13 +65,13 @@ func pushStudy(cmd *cobra.Command, opt *pushStudyOpt) {
...
 			return
 		}
 	} else if len(opt.args) > 0 {
 		err := json.Unmarshal(([]byte)(opt.args[0]), &in)
 		if err != nil {
-			log.Fatalf("Fail to Purse input: %v", err)
+			log.Fatalf("Fail to Parse input: %v", err)
```
Comment (gaocegege) [inline cmd/cli/command/push-study.go]: Same here

**Comment by gaocegege** [comment]
Thanks for your contribution!

LGTM :+1:

/lgtm
/approve

File: cmd/cli/command/push-study.go (hunk 3 above)
Comment (ytetra) [reply cmd/cli/command/push-study.go]: fixed

File: cmd/cli/command/push-study.go (hunk 4 above)
Comment (ytetra) [reply cmd/cli/command/push-study.go]: fixed

**Pull request merged**